docs/usage/configuration/config_documentation.md

```diff
@@ -0,0 +1,3464 @@
+## Welcome
```

Comment:
This could probably be changed into `Configuring Synapse` or something like that.